mods/ts/rules/defaults.yaml

```diff
@@ -668,6 +668,31 @@
 		Terrain: Tree
 	Tooltip:
 		Name: Tree
+	Health:
+		HP: 500
+	Armor:
+		Type: Wood
+	Targetable:
+		TargetTypes: Trees
+	WithDamageOverlay@SmallBurn:
+		DamageType: Incendiary
+		Image: burn-s
+		MinimumDamageState: Light
+		MaximumDamageState: Dead
+		Palette: effect
+	WithDamageOverlay@MediumBurn:
+		DamageType: Incendiary
+		Image: burn-m
+		MinimumDamageState: Medium
+		MaximumDamageState: Dead
```

Comment:
This makes all three fire overlays play at once when the tree reaches `Heavy` damage state.

The Maximum should be `Light` for `SmallBurn` and `Medium` here.